plugin/ochttp/propagation/tracecontext/propagation.go

```diff
@@ -98,13 +104,26 @@ func (f *HTTPFormat) SpanContextFromRequest(req *http.Request) (sc trace.SpanCon
 	return sc, true
 }
 
+func getRequestHeader(req *http.Request, name string, commaSeparated bool) (hdr string, ok bool) {
+	ok = false
```

Comment:
Remove, zero value of a boolean is false already.

Reply:
Got it, thanks.